### DISCUSSION TITLE
How to track UTM tracking as a dynamic variable?

### DISCUSSION THREAD

**Discussion by LLU27** [emailjs-com/help#45] [Q&A] [answered]
Hello,

I am currently trying to track the user_refererl link that the user is submitting the form. The form is set on the page "example.com", however if there is any UTM tracking for example like "https://www.example.com/utm_source=test&utm_medium=test&utm_id=test" ,
it is only coming in as "example.com" without the utm sources.

Is there any possible way to track the WHOLE link?

	const templateParams = {
		firstName: formData.firstName, 
		from_email: formData.email,
		from_phone: formData.phone,
		message: formData.message,
		utmSource: formData.utmSourceTrack,
		utmMedium: formData.utmMediumTrack,
		utmCampaign: formData.utmCampaignTrack,
	};
	
When I console log my templateParams,  I am able to see the whole object with the tracking information when testing, but in the Email.js template when i put {{utmSource}} , {{utmMedium}} etc...  it is empty when an email comes in

**Comment by xr0master** [comment, marked as answer]
Hi,
Yes, sure. You can add something like this
```js
const templateParams = {
  // your formData params
  absoluteURL: window.location.href,
};
```
